data/sql/updates/pending_db_world/rev_1714155297791279749.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Web Wrap: disable move, pacified; disable turn
+UPDATE `creature_template` SET `unit_flags` = 4 | 131076, `unit_flags2` = 32768 WHERE (`entry` = 16486);
+
+-- 28622: Web Wrap stunned dot
+DELETE FROM `spell_script_names` WHERE `spell_id` = 28622;
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`)
+VALUES(28622, 'spell_web_wrap_damage');
+
+-- 28618: pacify silence 5 seconds
+-- 28627: summon web wrap, morph id invisible
+DELETE FROM `spell_dbc` WHERE `ID` IN (28618, 28627);
```

Comment:
These spells already exist in the table, I would prefer an UPDATE query rather than a DELETE/INSERT.

Reply:
Ty, didn't notice they were in. I'll double check my work

Reply:
This did not work as expected..
```
    // Web Wrap
    ApplySpellFix({ 28618 }, [](SpellInfo* spellInfo)
    {
        spellInfo->Effects[EFFECT_0].Effect = 0;
        spellInfo->Effects[EFFECT_1].Effect = 0;
        spellInfo->DurationEntry = sSpellDurationStore.LookupEntry(36); // 5 seconds
    });
    ```
spell_dbc overwrites this I suppose

I left out 
```
UPDATE `spell_dbc` SET `EffectMiscValue_2` = 32785 WHERE `ID` = 28618; 
```
as `17286` gives invisibility with weapon showing.
`32785` is completely invisible with on weapon showing, found on a wrath mob's web wrap